### DISCUSSION TITLE
VS Code comment line keybinding stops working after opening copilot window once

### DISCUSSION THREAD

**Discussion by taylankasap** [community/community#118304] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Steps to reproduce:
1. Run "File: New Untitled Text File" command (`Ctrl+n`)
2. Change Language Mode to any language with comment line support (let's use HTML for now)
3. Press `Ctrl+/` (editor.action.commentLine keybinding) -> This will create a comment line
4. Press `Ctrl+Enter` to open Copilot Suggestions window
5. Press `Ctrl+w` to close Copilot Suggestions window
6. Press `Ctrl+/`

Expected:
`editor.action.commentLine` command should be invoked

Actual:
`github.copilot.acceptCursorPanelSolution` gets invoked

---

For some reason this when condition returns `true`, which causes this issue: `github.copilot.activated && github.copilot.panelVisible`

This happened on Linux, but I'm guessing this will happen on Windows too since it also uses Ctrl modifier.

**Comment by rohittcodes** [comment]
Hey @taylankasap, is your issue resolved?

I'm facing the same issue too. For me, it's on Windows; the expected command `editor.action.commentLine` isn't invoking. Instead, the action invokes `github.copilot.acceptCursorPanelSolution` whenever `github.copilot.activated` and `github.copilot.panelVisible` are active.

More info:
```
[KeybindingService]: / Received  keydown event - modifiers: [ctrl], code: Slash, keyCode: 191, key: /
[KeybindingService]: | Converted keydown event - modifiers: [ctrl], code: Slash, keyCode: 90 ('/')
[KeybindingService]: | Resolving ctrl+/
[KeybindingService]: \ From 5 keybinding entries, matched github.copilot.acceptCursorPanelSolution, when: github.copilot.activated && github.copilot.panelVisible, source: user extension GitHub.copilot.
[KeybindingService]: + Invoking command github.copilot.acceptCursorPanelSolution.
```

**Reply by taylankasap** [reply]
I cannot test at the moment. Let me get back to you on tuesday.

**Reply by gunzy83** [reply]
I have run into this issue as well (on Linux). It seems the editor thinks there is a panel open somewhere, but I can't for the life of me work out where. Closing an reopening the VSCode window is the only thing I have found that resolves the issue.

**Reply by gunzy83** [reply]
`Developer: Reload Window` works to get out of the bad state. Very annoying though.

**Reply by rohittcodes** [reply]
> Closing an reopening the VSCode window is the only thing I have found that resolves the issue.

The same worked for me on windows too. But the `Developer: Reload Window` option, won't let you do so. Hope they fix it soon.

**Reply by taylankasap** [reply]
I've updated to 1.88.1 and no, the issue is not resolved. I also reload the window as a workaround for now.

**Comment by jwikman** [comment]
A workaround that works for me (on Windows), to avoid reload VSCode now and then:
Change (or remove) Keyboard Shortcut for "GitHub Copilot: Accept panel suggestion at the cursor".

**Comment by artu-ole** [comment]
Same issue.
Someone filed a bug report already https://github.com/microsoft/vscode-copilot-release/issues/1143
Let's see where it goes.

**Comment by apatequil** [comment]
This doesn't resolve the source of the issue but if you want to work around this without actually removing the "GitHub Copilot: Accept panel suggestion at the cursor" you can update the when condition to:

github.copilot.activated && github.copilot.panelVisible && 1 == '2'

This will force the condition to fail 100% of the time. Once the underlying issue is fixed you can then just remove the additional condition and things should go back to working as expected.
